### PR TITLE
Ramquest actions uat connector certification

### PR DESCRIPTION
**Pull request by sanjev-ramquest**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by sanjev-ramquest** [comment]
> @sanjev-ramquest please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="RamQuest"]
> ```
> 
> > Options:
> > 
> > NA * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > RamQuest * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="RamQuest"
> > ```
> 
> Contributor License Agreement

Yes, I agree

**Comment by sanjev-ramquest** [comment]
I Agree



@microsoft-github-policy-service agree [company="RamQuest"]

Thanks ,
Sanjev Saalaivazan

**Comment by sanjev-ramquest** [comment]
@microsoft-github-policy-service agree company="RamQuest"

**Comment by sanjev-ramquest** [comment]
> @microsoft-github-policy-service agree [company="RamQuest"]

@microsoft-github-policy-service agree company="RamQuest"

File: certified-connectors/RamQuest Actions - UAT/Readme.md
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/Readme.md:1]: please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). All of the required headings should be present.

File: certified-connectors/RamQuest Actions - UAT/apiProperties.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiProperties.json]: please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json
Comment (vmanoharas) [inline certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

**Review by vmanoharas — COMMENTED** [review]
Hello @sanjev-ramquest, 

Kindly resolve review comments, and please don't merge the files until I approve the PR.

**Comment by sanjev-ramquest** [comment]
Thx @vmanoharas , Review changes resolved, Please rereview & approve

**Review by vmanoharas — APPROVED** [review]
Hello @sanjev-ramquest 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to Connector certification portal](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in [Connector certification portal](https://connectorcertification.azurewebsites.net/) . Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in [Connector certification portal](https://connectorcertification.azurewebsites.net/), please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**
